src/GitVersion.Core/VersionCalculation/VersionStrategies.cs

```diff
@@ -0,0 +1,13 @@
+namespace GitVersion.VersionCalculation;
+
+[Flags]
+public enum VersionStrategies
+{
+    None = 0,
+    ConfigNext = 1,
+    MergeMessage = 2,
+    TaggedCommit = 4,
+    TrackReleaseBranches = 8,
+    VersionInBranchName = 16,
+    TrunkBased = 32,
```

Comment:
Where did `Default` go?
```suggestion
    TrunkBased = 32,
    Default = ConfigNext | MergeMessage | TaggedCommit | TrackReleaseBranches | VersionInBranchName,
```

Reply:
We have no default

Reply:
Please forget the idea of trying to sum this up. This is the wrong place. We need to configure this semantically in the workspace template.

Reply:
Yeah, I agree. Moving this to `workflow` is a much better direction.